python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_network_services.schema.yml

```
@@ -1249,6 +1252,9 @@ $defs:
                       - 2
                       - 3
                     default: 2
+                  fast_leave:
+                    type: bool
+                    description: Enable IGMP snooping fast-leave feature on specified VLANs.
```

Actually sorry for missing this but here this is only for one given VLAN not VLANs
```suggestion
                    description: Enable IGMP snooping fast-leave feature.
```